src/vshaxe/Main.hx

```diff
@@ -42,6 +42,7 @@ function main(context:ExtensionContext) {
 	final problemMatchers = ["$haxe-absolute", "$haxe", "$haxe-error", "$haxe-trace"];
 	final api = {
 		haxeExecutable: haxeInstallation.haxe,
+		haxelibExecutable: haxeInstallation.haxelib.accessor,
```

Comment:
I introduced the class `ConfigurationAccessor<Config>` to not directly use the instance of `HaxelibExecutable`. It avoids changing the API and can later be used with `haxeExecutable` the same way (`haxeInstallation.haxe.accessor`).